reference/6/Microsoft.PowerShell.Core/About/about_Modules.md

```diff
@@ -284,44 +284,37 @@ a module does not uninstall the module. For more information, see
 
 ## Module and DSC Resource Locations, and PSModulePath
 
-The following are default locations for PowerShell modules. Starting in
-PowerShell 4.0, with the introduction of DSC, a new default module and DSC
-resource folder was introduced. For more information about DSC, see
-[about_DesiredStateConfiguration](/powershell/module/Microsoft.PowerShell.Core/About/about_DesiredStateConfiguration).
-
-- System: `$PSHOME\Modules` or
-  (`$env:windir\System32\WindowsPowerShell\v1.0\Modules`)
-  System modules are those that ship with Windows and PowerShell.
-
-  Starting in PowerShell 4.0, when PowerShell Desired State Configuration
-  (DSC) was introduced, DSC resources that are included with PowerShell are
-  also stored in `$PSHOME\Modules`, in the
-  `$PSHOME\Modules\PSDesiredStateConfiguration\DSCResources` folder.
+The `$env:PSModulePath` environment variable contains a list of folder
+locations that are searched to find modules and resources.
+
+The default locations assigned to `$env:PSModulePath` are:
 
-- Current user: `$HOME\Documents\WindowsPowerShell\Modules`
-  (`$env:UserProfile\Documents\WindowsPowerShell\Modules`)
+- System-wide locations: `$PSHOME\Modules`
 
-  or
+  These folders contain modules that ship with Windows and PowerShell.
 
-  `$HOME\My Documents\WindowsPowerShell\Modules`
-  (`$env:UserProfile\My Documents\WindowsPowerShell\Modules`)
+  DSC resources that are included with PowerShell are stored in the
+  `$PSHOME\Modules\PSDesiredStateConfiguration\DSCResources` folder.
 
-  This is the location for user-added modules prior to PowerShell 4.0.
+- User-specific modules: These are modules installed by the user in the user's
+  scope. `Install-Module` has a **Scope** parameter that allows you to specify
+  whether the module is installed for the current user or for all users. For
+  more information, see [Install-Module](../Install-Module.md).
 
-In PowerShell 4.0 and later releases of PowerShell, user-added modules and DSC
-resources are stored in `C:\Program Files\WindowsPowerShell\Modules`. Modules
-and DSC resources in this location are accessible by all users of the
-computer. This change was required because the DSC engine runs as local
-system, and could not access user-specific paths, such as
-`$home\Documents\WindowsPowerShell\Modules`.
+  The user-specific **CurrentUser** location on Windows is the
+  `PowerShell\Modules` folder located in the **Documents** location in your
+  user profile. The specific path of that location varies by version of Windows
+  and whether or not you are using folder redirection. By default, on Windows
+  10, that location is `$HOME\Documents\PowerShell\Modules`. On Linux or Mac,
```

Comment:
Why do you not want to use [Environment]::getFolderPath('myDocuments') which is what it is. Specifying some powershell will give people the wrong assumption